kollider-price/src/convert.rs

```diff
@@ -0,0 +1,23 @@
+use super::price_feed::error::KolliderPriceFeedError;
+use super::price_feed::KolliderPriceTicker;
+
+use shared::{
+    payload::{
+        ExchangeIdRaw, InstrumentIdRaw, KolliderBtcUsdSwapPricePayload, PriceMessagePayload,
+        PriceRatioRaw,
+    },
+    time::TimeStamp,
+};
+
+impl TryFrom<KolliderPriceTicker> for KolliderBtcUsdSwapPricePayload {
+    type Error = KolliderPriceFeedError;
+    fn try_from(value: KolliderPriceTicker) -> Result<Self, Self::Error> {
+        Ok(KolliderBtcUsdSwapPricePayload(PriceMessagePayload {
+            exchange: ExchangeIdRaw::from("Kollider"), // FIXME
```

Comment:
Please remove the `FIXME` comments before merge. Constants shouldn't be defined here.

Reply:
removed the FIXME and defined a constant